public_html/wp-content/mu-plugins/utilities/class-stripe-client.php

```diff
@@ -155,4 +155,56 @@ public static function get_fractional_unit_multiplier( $order_currency ) {
 
 		return $match;
 	}
+
+	/**
+	 * Convert an amount in the currency's base unit to its equivalent fractional unit.
+	 *
+	 * Stripe wants amounts in the fractional unit (e.g., pennies), not the base unit (e.g., dollars).  The data
+	 * here comes from https://stripe.com/docs/currencies.
+	 *
+	 * Note: This uses different data than `get_fractional_unit_multiplier` above, these have different data
```

Comment:
For consistency with our other code comments that expect a followup, I'd change `Note` to `@todo`

Reply:
Feel free to use the suggestion functionality for that, then I can just commit it.

```suggestion
	 * @todo This uses different data than `get_fractional_unit_multiplier` above, these have different data
```